submissions/krylenger/memory-pair-game/app.js

```diff
@@ -0,0 +1,244 @@
+            break;
+        case 1:
+        case 2:
+        case 7:
+        case 8:
+            newCardFlipperFront.classList.add('card__flipper-front', card.key, `card--vertical-line-img`);
+            break;
+        case 3:
+            newCardFlipperFront.classList.add('card__flipper-front', card.key, `card--left-corner-img`);
+            break;
+        case 6:
+            newCardFlipperFront.classList.add('card__flipper-front', card.key, `card--right-corner-img`);
+            break;
+        case 4:
+        case 5:
+            newCardFlipperFront.classList.add('card__flipper-front', card.key, `card--horizontal-line-img`);
+            break;
+        default:
+            newCardFlipperFront.classList.add('card__flipper-front', card.key, `card--clef-img`);
+    }
+    
+}
+
+const createNewCard = (card, cardNumber) => {
+    console.log(cardNumber);
+    const newCard = document.createElement('div');
+    const newCardFlipper = document.createElement('div');
+    const newCardFlipperFront = document.createElement('div');
+    const newCardFlipperBack = document.createElement('div');
+    if (card.musicCard) {
+        newCard.classList.add('card', `card--${cardNumber}`, 'musicCard')
+        newCardFlipperBack.classList.add(`card--music-img`);
+    } else {
+        newCard.classList.add('card', `card--${cardNumber}`);
+        newCardFlipperBack.classList.add(`card--${card.text}-img`);
+    }
+    newCardFlipper.classList.add('card__flipper');
+    newCardFlipper.setAttribute('ontouchstart', "this.classList.toggle('card__flipper--flip');")
+    styleNewCardFlipperFront(newCardFlipperFront, card, cardNumber);
+    newCardFlipperBack.classList.add('card__flipper-back');
+    newCard.appendChild(newCardFlipper);
+    newCardFlipper.appendChild(newCardFlipperFront);
+    newCardFlipper.appendChild(newCardFlipperBack);
+    return newCard;
+}
+
+const shuffleArr = (arr) => {
+    return arr.sort(() => 0.5 - Math.random());
+}
+
+const loadCards = (cardsArr) => {
+    const shuffledCards = shuffleArr(cardsArr);
+    const fragment = document.createDocumentFragment();
+    shuffledCards.forEach((card, index) => {
+        const newCard = createNewCard(card, index);
+        fragment.appendChild(newCard);
+    })
+    mainGridContainer.appendChild(fragment);
+}
+
+const recreateMainGridContainer = () => {
+    mainGridContainer.remove();
+    mainGridContainer = document.createElement('div');
+    mainGridContainer.classList.add('main__grid-container');
+    main.appendChild(mainGridContainer);
+}
+
+const resetGame = () => {
+    recreateMainGridContainer();
+    loadCards(cards);
+    guessedCards = [];
+    playedSongs = [];
+    handleClick();
+}
+
+const handlePlayingCards = (cardStatus) => {
+    const playingNow = document.querySelectorAll(".playingNow");
+    playingNow.forEach((card) => {
+        card.classList.remove("playingNow");
+        if (cardStatus === 'guessed') {
```

Comment:
you can create a constant for this status because I saw it several times in your code

Reply:
Thanks for pointing out to this! 
I've researched more and now understand why it's better change literals to constants!
✅ Done

Reply:
When you'll use TypeScript you can use enums for storing all values like this. It's quite useful and easy. But here constants are nice decision as well